Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/d/Library/Python/3.11/lib/python/site-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
qui {
matrix b = e(b)
matrix list b
//SV -> scenario vector (i.e., user-input = scenario)
//In the scenario below b0=1, b1=0, b2=0, b3=0, b4=0m b5=0
matrix define SV = (1, 0, 0, 0, 0) 
//yields the logHR
matrix loghr = SV*b'
use "${repo}s0.dta", clear
g f0 = (1 - s0)*100
g f1 = f0*exp(loghr[1,1])
noi sum f1 if inrange(_t,0,.0001)
noi sum f1 if inrange(_t,4.9999,5.0001)
noi sum f1 if inrange(_t,9.9999,10.0001)
noi sum f1 if inrange(_t,14.9999,15.0001)
noi sum f1 if inrange(_t,19.9999,20.0001)
//Segev, JAMA, 2010
noi di "Donor mortality at 12-year follow-up:" %3.1f r(max) "%" 
#delimit;
line f1 _t if inrange(_t,1,15), 
    sort 
	connect(step step) 
	ylab(0(10)40, 
	    format(%2.0f)
	) 
	xlab(0(3)15)
	yti("%")
	xti("Years")
	;
#delimit cr
}
------------------

----- stdout -----

file https://github.com/jhustata/project/raw/main/s0.dta not found
----- stderr -----
r(601);
----- stderr -----
r(601);
----- stdout -----

------------------



